lib/py/src/transport/THttpClient.py

```diff
@@ -100,7 +100,7 @@ def basic_proxy_auth_header(proxy):
         ap = "%s:%s" % (urllib.parse.unquote(proxy.username),
                         urllib.parse.unquote(proxy.password))
         cr = base64.b64encode(ap.encode()).strip()
-        return "Basic " + cr
+        return "Basic " + cr.decode()
```

Comment:
this is a python 2 vs. 3 issue. the old code works in python 2 and the new code works in python 3. currently we still officially support both so you probably need to find a way to make it work in both 2 and 3 (maybe `six` can help in this case?)

we do have a plan to drop python 2 support, so that won't be needed after we do that. but we just don't have the time to actually work on that yet (you are welcomed to take that on :) https://issues.apache.org/jira/browse/THRIFT-5537)

Reply:
Ah, that makes sense. I'll see if I can make this work using `six`.